examples/lines_bars_and_markers/errorbar_subsample.py

```diff
@@ -12,22 +12,29 @@
 
 # example data
 x = np.arange(0.1, 4, 0.1)
-y = np.exp(-x)
+y = np.exp(np.vstack([-1.0 * x, -.5 * x]))
```

Comment:
I think, it's easier to follow if you keep the data in two separate variables:
~~~
y1 = np.exp(-x)
y2 = np.exp(-0.5 * x)
~~~